openlibrary/templates/lists/widget.html

```diff
@@ -37,6 +37,7 @@
         d = storage({
             "name": list.name or "",
             "key": list.key,
+            "last_modified": list.last_modified.isoformat(),
```

Comment:
Can't the dates be used directly for sorting without conversion to strings since they're already comparable?

Reply:
To answer my own question, no, because it's trying to create a Storage object which has to be JSONisable. Having said that, none is necessary since `user.get_lists(sort=True)` can be used instead of the current `user.get_lists(sort=False)`.